vc_vidyo/indico_vc_vidyo/forms.py

```diff
@@ -7,19 +7,23 @@
 
 from __future__ import unicode_literals
 
+import re
+
 from wtforms.fields.core import BooleanField
-from wtforms.fields.simple import TextAreaField
+from wtforms.fields.simple import StringField, TextAreaField
 from wtforms.validators import DataRequired, Length, Optional, Regexp, ValidationError
 
 from indico.modules.vc.forms import VCRoomAttachFormBase, VCRoomFormBase
 from indico.web.forms.base import generated_data
 from indico.web.forms.fields import IndicoPasswordField, PrincipalField
+from indico.web.forms.validators import IndicoRegexp
 from indico.web.forms.widgets import SwitchWidget
 
 from indico_vc_vidyo import _
 from indico_vc_vidyo.util import iter_user_identities, retrieve_principal
 
 
+ROOM_NAME_RE = re.compile(r'[\w\-]+')
```

Comment:
Since `IndicoRegexp(ROOM_NAME_RE)` runs at import time as well, there's not much benefit in precompiling the regex instead of just passing the string directly to `IndicoRegexp()`

Reply:
I just moved it around. But yeah, I'll do that.